cypress/integration/mainView_spec.js

```diff
@@ -0,0 +1,51 @@
+describe('Show landing page of Head To Toe', () => {
+
+  beforeEach(() => {
+    cy.fixture('mockDr.json')
+      .then(mockData => {
+        cy.intercept('GET', 'https://head-to-toe-be.herokuapp.com/api/v1/medical_professionals?type=doctor&state=South Dakota', {
+          statusCode: 201,
+          delay: 100,
+          body: mockData
+        })
+      })
+    cy.visit('http://localhost:3000/')
+  });
+
+  it('Should be able to open to the main page', () => {
+  cy.url().should('eq', 'http://localhost:3000/')
+  });
+
+  it('Should be show a background image covering the whole page', () => {
+    cy.get('.App').should('have.css', 'background-image', 'url("http://localhost:3000/static/media/peacefulOceanImage.dfb3c4eb.png")')
+  });
+
+  it('Should display the title on the main page', () => {
+  cy.get('.App-title').should('be.visible', 'Head To Toe')
+    .get('.App-title').should('contain', 'Head To Toe')
+    .get('.App-title').should('have.css', 'font-family', 'opendyslexic')
+  });
+
+  it('Should display the form, dr and mhp buttons for user', () => {
+    cy.get('.App-buttons').should('be.visible')
+      .get('.App-buttons').should('contain', 't')
+      .get('.App-buttons').should('have.css', 'font-family', 'opendyslexic')
+  });
```

Comment:
Having all the buttons contain 't' doesn't really seem to confirm all of them. I think we'll probably want their own test to make sure each button exists how we want it.

Reply:
Same answer as above, adding the already existing testing that we have to this test that tests for css or moving its css testing to those tests are two solutions to the minimum value of testing for the population of singular letter.